testing/vcs/test_boxfill_number_color_more_than_number_levels.py

```diff
@@ -34,6 +34,7 @@
     failed = True
 except:
     failed = False
+    raise
 
```

Comment:
I'd prefer leaving the blank `raise`, as this reraises the previous exception and preserves the stack trace of the actual error.